File: Algorithm/Jungol/j5932/j5932.py
# 숫자사각형 4-2
# https://www.jungol.co.kr/problem/5932

inputVal = int(input())

for i in range(0, inputVal):
    for j in range(0, inputVal):
        if (i+1)%2 != 0:
            print(j+1, end=' ')
        else:
            print(inputVal-j, end=' ')
    print()

File: Algorithm/Jungol/j5932/j5932.c
// 숫자사각형 4-2
// https://www.jungol.co.kr/problem/5932

#include <stdio.h>

int main(void) {
    int inputVal;
    scanf("%d", &inputVal);

    for(int i=0; i<inputVal; i++) {
        for(int j=0; j<inputVal; j++) {
            if((i+1)%2 != 0) {
                printf("%d ", j+1);
            }
            else {
                printf("%d ", inputVal-j);
            }
        }
        printf("\n");
    }

    return 0;
}